src/app/configureApp.jsx

```diff
@@ -1,71 +1,70 @@
-import React from 'react';
-import ApplicationMenu from './ApplicationMenu';
+import RoutingMenu from 'terra-application-layout/lib/menu/RoutingMenu';
 import Components from './components/Components';
-import ComponentsMenu from './components/ComponentsMenu';
 import Home from './components/Home';
 
-const injectConfig = configuredProps => (
-  Component => (
-    props => (
-      <Component {...props} {...configuredProps} />
-    )
-  )
-);
-
 const buildComponent = (Component, configuredProps) => (
   {
     default: {
-      componentClass: injectConfig(configuredProps)(Component),
+      componentClass: Component,
+      props: configuredProps,
```

Comment:
Eventually we will probably want to open these config to allow props to be passed with the component that we combine with ours

Reply:
Yeah I was thinking about adding that, but we don't have a case for it yet.

Reply:
In working on the docs for dev-site I noticed it with the menu, but we've removed the menu here so I think its fine to leave until there is need!